src/directLine.ts

```diff
@@ -252,6 +261,7 @@ export interface IBotConnection {
     connectionStatus$: BehaviorSubject<ConnectionStatus>,
     activity$: Observable<Activity>,
     end(): void,
+    getConversation(): Conversation,
```

Comment:
Speech just needs the `referenceGrammarId` property, which is currently part of Conversation. Just didn't want to create another abstraction.